readme.md

```diff
@@ -71,78 +71,10 @@ AutoRest needs the below config to pick this up as a plug-in - see https://githu
 
 #### Javagen
 
-``` yaml !$(fluent)
-use-extension:
-  "@autorest/modelerfour": "4.8.221"
-use: $(this-folder)//javagen
-
-pipeline:
-
-# --- extension remodeler ---
-
-  # "Shake the tree", and normalize the model
-  modelerfour:
-    input: openapi-document/multi-api/identity     # the plugin where we get inputs from
-    flatten-models: true
-    flatten-payloads: true
-    group-parameters: true
-  
-  # allow developer to do transformations on the code model.
-  modelerfour/new-transform:
-    input: modelerfour
-
-  preprocessor:
-    input: modelerfour/identity
-
-  javagen:
-    input: preprocessor
-    output-artifact: java-files
-  
-  javagen/emitter:
-    input: javagen
-    scope: scope-javagen/emitter
-
-scope-javagen/emitter:
-    input-artifact: java-files
-    output-uri-expr: $key
-
-output-artifact: java-files
+``` yaml $(java) && !$(fluent)
+use: $(this-folder)/javagen
 ```
 
-``` yaml $(fluent)
-use-extension:
-  "@autorest/modelerfour": "4.9.236"
+``` yaml $(java) && $(fluent)
 use: $(this-folder)/fluentgen
```

Comment:
It seems easier to only load javagen or fluentgen here. Since javagen will then load preprocessor, 2 readme from them would complete the pipeline.